pkg/deleter/deleter.go

```diff
@@ -266,6 +267,11 @@ func (d *Deleter) cleanFilePV(pv *v1.PersistentVolume, mountPath string, config
 	klog.Infof("Deleting PV file volume %q contents at hostpath %q, mountpath %q", pv.Name, pv.Spec.Local.Path,
 		mountPath)
 
+	if runtime.GOOS == "windows" {
+		// the path to use is the one sent to CSI Proxy and must be in the context of the host
+		return d.VolUtil.DeleteContents(pv.Spec.Local.Path)
```

Comment:
maybe add some details about this logic?

Reply:
I added another sentence trying to clarify it